src/index.ts

```diff
@@ -68,11 +68,19 @@ async function monitor(): Promise<void> {
 }
 
 async function setupSysdigIntegration(): Promise<void> {
-  if (!config.SYSDIG_ENDPOINT || !config.SYSDIG_TOKEN) {
-    logger.info({}, 'Sysdig integration not detected');
+  if (
+    !(
+      config.SYSDIG_REGION_URL &&
+      config.SYSDIG_API_TOKEN &&
+      config.SYSDIG_CLUSTER_NAME
+    )
+  ) {
+    logger.info({}, 'Sysdig not properly configured');
```

Comment:
this will be logged for all customers not using the sysdig integration so we could do something like this:
```
  if (
    !config.SYSDIG_REGION_URL &&
    !config.SYSDIG_API_TOKEN &&
    !config.SYSDIG_CLUSTER_NAME
  ) {
    console.log('Sysdig integration not detected');
    return;
  }
  if (
    !config.SYSDIG_REGION_URL ||
    !config.SYSDIG_API_TOKEN ||
    !config.SYSDIG_CLUSTER_NAME
  ) {
    console.log('Sysdig integration not configured correctly');
    return;
  }
  console.log('Sysdig integration configured correctly');
  return;
```
to differentiate between the 3 states

alternatively, we can make the secret values not optional